src/Line3d.ts

```diff
@@ -33,69 +32,11 @@ export default class Line3d implements IDrawableObject {
     // artifacts can occur where contours are drawn around lines. This layer
     // (MESH_NO_PICK_OCCLUSION_LAYER) does not occlude/interact with the pick
     // buffer but still writes depth information for the volume.
+    this.meshPivot.layers.set(MESH_NO_PICK_OCCLUSION_LAYER);
```

Comment:
do children inherit their parent's layers? (it looks like the following line does the same to children of meshPivot)